src/components/interactive-map.js

```diff
@@ -90,14 +90,14 @@ export default class InteractiveMap extends PureComponent {
           ref: map => {
             this._map = map;
           },
-          children: mapVisible && this.props.children
+          children: null
         })),
         createElement('div', {
-          key: 'map-children',
-          style: {position: 'absolute', left: 0, top: 0}
-        },
-          !mapVisible && this.props.children
-        )
+          key: 'map-overlays',
+          className: 'overlays',
```

Comment:
Did this class name need to be synched with static-map? Should we add comments in both places to that effect?